docs/usage.md

```diff
@@ -5,8 +5,9 @@
 
 # Usage
 
-The **FortiAI** solution pack offers a variety of functions through its **AI Assistant** to managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
+The **FortiAI** solution pack offers a variety of functions for managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
```

Comment:
variety of functions or variety of feature?

Reply:
Functions. Not in the code sense. In general, non-techy, users' perspective